peak_finder/driver.py

```diff
@@ -348,12 +380,17 @@ def run(self):  # pylint: disable=R0912, R0914, too-many-statements
 
                     params = TrendLineParameters.build(inputs)
                     driver = TrendLinesDriver(params)
-                    out_trend = driver.make_curve()
 
-                    if out_trend is not None:
-                        driver.add_ui_json(out_trend)
+                    try:
+                        out_trend = driver.make_curve()
+
+                        if out_trend is not None:
+                            driver.add_ui_json(out_trend)
+
+                    except QhullError as e:
+                        print(f"Warning - Skipping Trend Lines! ! ! \n{e}")
```

Comment:
That might be a bit obscure to the user.  Can you give more information about why this error occurs.

Reply:
Optional: Switch to logger.info